assembl/static2/js/app/graphql/QuestionPostsQuery.graphql

```diff
@@ -0,0 +1,21 @@
+query PostsQuery($id: ID!, $first: Int!, $after: String!) {
```

Comment:
Please rename the query to QuestionPosts (the same name as the filename without Query suffix)

Reply:
done